wweval/R/plots.R

```diff
@@ -1,3 +1,122 @@
+#' Compare growth rates in wastewater and hospital admissions data
+#'
+#' @param input_hosp_data tibble of hospital admissions counts for a
+#' particular location and forecast date
+#' @param input_ww_data tibble of wastewater concentrations for a particular
+#' location and forecast date
+#' @param location string indicating the location of interest
+#' @param forecast_date string indicating the forecast date of interest
+#' @param rate string indicating whether the plot should be a daily or
+#' weekly growth rates
+#'
+#' @return a ggplot object containing site-lab level exponential growth rates
+#' with the state level hospital admissions growth rates overlaid
+#' @export
+get_growth_rate_plot <- function(input_hosp_data,
+                                 input_ww_data,
+                                 location,
+                                 forecast_date,
+                                 rate = "daily") {
+  hosp_data <- input_hosp_data |> dplyr::mutate(
+    admits_7d_rolling = zoo::rollmean(daily_hosp_admits,
+      k = 7, na.pad = TRUE,
+      align = "center"
```

Comment:
I can imagine wanting to try a different align option (for example in real time analysis). Might be worth making the align a function argument.